.travis.yml

```diff
@@ -1,24 +1,33 @@
 language: python
-matrix:
-  include:
-    - python: 2.7
-      env: SYMPY_VERSION="oldest"
-    - python: 2.7
-      env: SYMPY_VERSION="latest"
-    - python: 2.7
-      env: SYMPY_VERSION="master"
-virtualenv:
-  system_site_packages: true
+env:
+  matrix:
+  - SYMPY_VERSION="latest"
+  - SYMPY_VERSION="master"
+  - SYMPY_VERSION="oldest"
+python:
+  - 2.7
+  - "2.7_with_system_site_packages"
 before_install:
-  - sudo apt-get update
+  - sudo apt-get update -qq
   # Remove the version of NumPy pre-installed into the Travis virtualenv so
   # that we test with the system installed version.
-  - pip uninstall -y numpy
+  - pip uninstall -y numpy;
+  - if [[ $TRAVIS_PYTHON_VERSION != "2.7_with_system_site_packages" ]]; then
+      sudo apt-get install -qq libatlas-dev libatlas-base-dev liblapack-dev gfortran;
+      wget http://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh -O miniconda.sh;
+      chmod +x miniconda.sh;
+      ./miniconda.sh -b;
+      export PATH=/home/travis/miniconda/bin:$PATH;
```

Comment:
You probably don't want to install conda at all if you want to use the apt packages, like the previous setup. Right here you skip over the virtualenv with this line.